walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
   </head>
```

Comment:
Is this file required ?  If not, delete it

Reply:
This file was part of the demo provided to us by SPS to help us get started. It is not part of my portfolio directory. It is in the walkthroughs directory. So it is not required.